spyder/widgets/editor.py

```diff
@@ -274,6 +274,70 @@ def breakpoints_changed(self):
             self.save_breakpoints.emit(self.filename, repr(breakpoints))
 
 
+class TabSwitcherWidget(QListWidget):
+    """Show tabs in mru order and change between them."""
+    def __init__(self, parent, stack_history, tabs):
+        QListWidget.__init__(self, parent)
+        self.setWindowFlags(Qt.SubWindow | Qt.FramelessWindowHint)
+
+        self.editor = parent
+        self.stack_history = stack_history
+        self.tabs = tabs
+
+        self.setSelectionMode(QListWidget.SingleSelection)
+        self.itemActivated.connect(self.item_selected)
+
+        self.id_list = []
+        self.load_data()
+        size = CONF.get('main', 'completion/size')

```

Comment:
I reused completion widget size widget, is that ok? should I add a setting? or leave a fixed size